datasource/src/main/resources/liquibase/changelog/changelog-0.1.xml

```diff
@@ -65,4 +65,26 @@
         </rollback>
     </changeSet>
 
+    <changeSet id="create_run_table" author="manuel_vigueras">
+        <createTable tableName="run">
+            <column name="run_id" type="${uuid.type}"/>
+            <column name="project_id" type="${uuid.type}"/>
+            <column name="project_version" type="${text.type}"/>
+            <column name="status" type="${text.type}"/>
+            <column name="time_start" type="${timestamp.type}"/>
+            <column name="time_end" type="${timestamp.type}"/>
+            <column name="user_id" type="${uuid.type}"/>
+            <column name="results" type="${text.type}"/>
+            <column name="cmwl_workflow_id" type="${text.type}"  defaultValueBoolean="true"/>
+        </createTable>
+        <addPrimaryKey constraintName="run_pk" tableName="run" columnNames="run_id"/>
+        <addForeignKeyConstraint constraintName="fk_run_user" baseTableName="run" baseColumnNames="user_id" referencedTableName="user" referencedColumnNames="user_id"/>
+        <addForeignKeyConstraint constraintName="fk_run_project" baseTableName="run" baseColumnNames="project_id" referencedTableName="project" referencedColumnNames="project_id"/>
+        <addNotNullConstraint constraintName="project_version_nn" tableName="run" columnName="project_version"/>
+        <addNotNullConstraint constraintName="status_nn" tableName="run" columnName="status"/>
+        <addNotNullConstraint constraintName="time_start_nn" tableName="run" columnName="time_start"/>
+        <addNotNullConstraint constraintName="results_nn" tableName="run" columnName="results"/>
+        <addNotNullConstraint constraintName="cmwl_workflow_id_nn" tableName="run" columnName="cmwl_workflow_id"/>
```

Comment:
Cromwell workflow id should be nullable (in code and database)

Reply:
Change applied it